client/idm/console/src/main/java/org/apache/syncope/client/console/status/AnyStatusDirectoryPanel.java

```diff
@@ -378,7 +378,7 @@ protected List<StatusBean> getStatusBeans(final long first, final long count) {
                                     actual instanceof GroupTO);
                             statusBean.setLinked(false);
                             return statusBean;
-                        }).collect(Collectors.toList()));
+                        }).toList());
```

Comment:
Switching from

```java
collect(Collectors.toList())
```

to

```java
toList()
```

with Wicket should be carefully considered, as the given list object could be used as `Model` for a `Component` and required to be mutable.

Reply:
@mmoayyed have you had the chance to test via browsers the changes in Console and Enduser? I am afraid the current test suite does not cover all scenarios.

Reply:
I ran some brief smoke tests, but will do some more thorough testing today.

Reply:
Thank you!

Reply:
I have run a set of tests against Syncope console and end-user, and so far I have not seen any problems. Creating realms, users, etc and various operations on users are work OK. I have also passed through various screens in WA and they too appear to work normally.

Reply:
Ok thanks.
The checks are needed especially with the operations that are involving the changed classes, so for example searching for Users, Groups or Any Objects, merging Linked Accounts, perform some Reconciliation etc.

Any way, let's go ahead and merge, just keep in mind that some immutable-related issues might arise and we know where they are from.